nautobot_firewall_models/tests/test_models.py

```diff
@@ -285,6 +285,33 @@ def test_policy_to_json(self):
         self.assertEqual(json_details["destination_services"][0]["port"], "53")
 
 
+class TestNATPolicyRuleModels(TestCase):
+    """Test the NATPolicyRule model."""
+
+    def setUp(self) -> None:
```

Comment:
I like type hints but only one in the code isn't much value. We can come back in a later PR to add them :-)

```suggestion
    def setUp(self):
```

Reply:
I copied the test case down from one of the existing ones - I can remove this if you want but its an existing pattern.